wct.docker.conf.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
al die test-config ne keer in de util steken staat ook op de nabije planning?

Reply:
Die zit al in de vl-ui-util, wordt toegevoegd en aangepast via die weg cfr. `install:copy:wct`.